dirty_cat/tests/test_target_encoder.py

```diff
@@ -193,7 +207,21 @@ def _test_missing_values(input_type, missing):
         return
 
 
-def _test_missing_values_transform(input_type, missing):
+@pytest.mark.parametrize(
+    "input_type, missing",
+    [
+        ("list", "aaa"),
+        ("numpy", "error"),
+        ("pandas", ""),
+        ("list", "error"),
+        ("numpy", ""),
+        ("pandas", "aaa"),
+        ("list", ""),
+        ("numpy", "aaa"),
+        ("pandas", "error"),
+    ],
```

Comment:
Instead of manually writing the list, you could use [`itertools.product`](https://docs.python.org/3.8/library/itertools.html#itertools.product)
Something along the lines of
```suggestion
    product(["list", "pandas", "numpy"], ["", "aaa", "error"])
```